mindsdb/integrations/handlers/airtable_handler/connection_args.py

```diff
@@ -4,23 +4,15 @@
 
 
 connection_args = OrderedDict(
-    base_id={
-        'type': ARG_TYPE.STR,
-        'description': 'The Airtable base ID.'
+    base_id={"type": ARG_TYPE.STR, "description": "The Airtable base ID."},
```

Comment:
We should add if each key is required or not as :
```
        'required': False
```

Reply:
Alright, done!